### ISSUE TITLE
FanZoneExhaust MaximumFlowRate error

### ISSUE THREAD

**Issue by MatthewSteen** [NREL/openstudio-standards#836]
New FanZoneExhaust objects don't have a MaximumFlowRate by default, which causes the `Create Baseline Building` measure to fail during SR2 on the `maximumFlowRate` call on L86. Suggest adding a trap for this condition or changing the default in the OS App/API.

https://github.com/NREL/openstudio-standards/blob/eb37ff03903a2a500b44fae2a5b7eaeeef5c36ab/lib/openstudio-standards/standards/Standards.Fan.rb#L76-L87

![image](https://user-images.githubusercontent.com/8397034/84958248-39e5f980-b0ba-11ea-83a8-8d83a926ad4f.png)

**Comment by carlobianchi89** [comment]
@MatthewSteen I am trying to reproduce the error to fix this. create_baseline_building_test.rb run the measure and does not fail. Could you point me to an example in which fan_fanpower fails?

**Comment by MatthewSteen** [comment]
@carlobianchi89 I don't have one at my fingertips. The `create_baseline_building_test.rb` must not have a `FanZoneExhaust` object _without_ a maximum flow rate because the issue is the `maximumFlowRate.get` call on line 86. 

The code below illustrates the issue, but you could also (1) create a DOE prototype building, (2) add a FanZoneExhaust to a zone, and (3) apply the `Create Baseline Building` measure. 

```
require 'openstudio'
include OpenStudio::Model

m = Model.new
f = FanZoneExhaust.new(m)
puts m
f.maximumFlowRate.get
```

```
OS:Version,
  {b0222d1b-c55e-4a9e-8e8a-e39ec58280a1}, !- Handle
  2.9.0;                                  !- Version Identifier

OS:Fan:ZoneExhaust,
  {9e7afddb-8843-44a2-9636-539d78e976e4}, !- Handle
  Fan Zone Exhaust 1,                     !- Name
  ,                                       !- Availability Schedule Name
  0.6,                                    !- Fan Total Efficiency
  0,                                      !- Pressure Rise {Pa}
  ,                                       !- Maximum Flow Rate {m3/s}
  ,                                       !- Air Inlet Node Name
  ,                                       !- Air Outlet Node Name
  General,                                !- End-Use Subcategory
  ,                                       !- Flow Fraction Schedule Name
  Decoupled;                              !- System Availability Manager Coupling Mode

6:in `get': Optional not initialized (RuntimeError)
from -:6:in `<main>'
```

**Comment by mdahlhausen** [comment]
Fixed with #875
